ambry-protocol/src/main/java/com.github.ambry.protocol/PutRequest.java

```diff
@@ -57,8 +57,7 @@
   private static final short Put_Request_Version_V2 = 2;
```

Comment:
you could remove this declaration if the frontends are first updated and then the servers? or are you planning to do cleanup in one shot?